src/woo_publications/publications/api/serializers.py

```diff
@@ -72,6 +74,33 @@ class Meta:  # pyright: ignore
             "laatst_gewijzigd_datum",
             "bestandsdelen",
         )
+        extra_kwargs = {
+            "uuid": {
+                "read_only": True,
+            },
+            "publicatiestatus": {
+                "help_text": _(
+                    "\n**Disclaimer**: you can't create a {} document."
+                ).format(PublicationStatusOptions.revoked.label.lower())
+            },
+        }
+
+    def validate(self, attrs):
+        self.instance: Document
+
+        if self.context["request"].method == "POST":
```

Comment:
this isn't very elegant, I think better would be:

`is_create = self.instance is None` or if the instance is set: `is_create = self.instance.pk is None`